Write a cromwell test. (#40)
LUMC/pytest-workflow: docs/manual.rst

      stderr:
        contains:
         - "(100%) done"

WDL with Cromwell example
-------------------------

An example yaml file that could be used to test a WDL pipeline that is run with

> Cromwell is explained below.

Reviewer: This sentence is a bit hard to follow. Maybe something like this would be better:
"Below an example yaml file is explained, which can be used to test a WDL pipeline run through Cromwell."

One problem with Cromwell is the way it handles relative paths and how it
handles the input file:

+ Relative paths are written only within the ``cromwell-executions`` folder

>   if you want to write outside this folder you need absolute paths. This is

Reviewer: I feel like it `If you want...` should be the start of a new sentence.

  fine but for testing your pipeline ``pytest-workflow`` creates a temporary

>   folder from which the pipeline is run. You don't know before which path this

Reviewer: ```suggestion
  folder from which the pipeline is run. You don't know beforehand which path this
```

>   is, but you could use environment variable ``$PWD``.

Reviewer: ```suggestion
  is, but you could use the environment variable ``$PWD``.
```

> + However problem two is that inputs can only be supplied to Cromwell in a

Reviewer: ```suggestion
+ However, the second problem is that inputs can only be supplied to Cromwell in a
```
